test/AWS.Deploy.Orchestration.UnitTests/DeployedApplicationQueryerTests.cs

```diff
@@ -471,8 +471,38 @@ public async Task GetPreviousSettings_BeanstalkEnvironment()
             Assert.Equal("false", optionSettings[Constants.ElasticBeanstalk.XRayTracingOptionId]);
         }
 
-        [Fact]
-        public async Task GetPreviousSettings_BeanstalkWindowsEnvironment()
+        [Theory]
+
+        [InlineData(@"{
+              ""manifestVersion"": 1,
+              ""deployments"": {
+                ""aspNetCoreWeb"": [
+                  {
+                    ""name"": ""app"",
+                    ""parameters"": {
+                      ""iisPath"": ""/path"",
+                      ""iisWebSite"": ""Default Web Site Custom""
+                    }
+                  }
+                ]
+              }
+            }")]
+        [InlineData(@"{
+              ""manifestVersion"": 1,
+              // comments
```

Comment:
Can you assert that we're not deleting the user's comments when we write it back out?